tasks/aiida-prepare.yml

```diff
@@ -53,32 +53,36 @@
     password: "{{ aiida_postgres_db_pw }}"
     encrypted: true
 
-- name: "Let virtualenvwrapper use the correct python version"
-  lineinfile:
-    path: "${HOME}/.bashrc"
-    regexp: "export VIRTUALENVWRAPPER_PYTHON=.*"
-    line: "export VIRTUALENVWRAPPER_PYTHON={{ ansible_python_interpreter }}"
-  when: ansible_python_interpreter is defined
+- name: "create python{{ aiida_python_version }} virtual env: {{ aiida_venv }}"
+  import_role:
+    name: marvel-nccr.python
+  vars:
+    python_base_version: "{{ aiida_python_version }}"
+    python_install_dev: true  # required to install pymatgen
+    python_install_tk: true  # required by ase.gui
```

Comment:
ah hm, I see this is in the python role...
that's a rather specific thing to put in a general python role, but ok

Reply:
why? this is to install the bindings specifically for this python version, i.e. `apt install python3.7-tk`